src/param.zig

```diff
@@ -5,6 +5,7 @@ const Ip4Address = std.os.linux.sockaddr;
 const Params = struct {
     addr: u32 = undefined,
     subnetmask: u32 = undefined,
+    gateway: u32 = undefined,
```

Comment:
Should we set default value rather than `undefined` ?
Or insert assertion before calling lwip?

Reply:
How about make it nullable and the default value null?

Reply:
Looks good!